order/views.py

```diff
@@ -120,13 +124,70 @@ def delete(self, request, *args, **kwargs):
         except OrderStatus.DoesNotExist:
             return JsonResponse({'message': 'BAD_REQUEST'}, status=400)
 
+class PaymentView(View):
+    @login_decorator
+    def get(self, request, *args, **kwargs):
+        try:
+            user         = request.user
+            order        = Order.objects.get(user=user, status__name=SHOPPING_BASKET)
+            cart_lists   = order.cart_set.all()
+            product_info = [
+                {
+                    "cartId"    : cart_list.id,
+                    "productId" : cart_list.product_id,
+                    "product"   : cart_list.product.name,
+                    'option'    : cart_list.option,
+                    'quantity'  : cart_list.quantity,
+                    'totalPrice': cart_list.total_price,
+                    'eachPrice' : cart_list.product.price,
+                    'urlImage'  : cart_list.product.image_url,
+                } for cart_list in cart_lists
+            ]
```

Comment:
```suggestion
            product_info = [
                {
                    "cartId"    : cart_list.id,
                    "productId" : cart_list.product_id,
                    "product"   : cart_list.product.name,
                    'option'    : cart_list.option,
                    'quantity'  : cart_list.quantity,
                    'totalPrice': cart_list.total_price,
                    'eachPrice' : cart_list.product.price,
                    'urlImage'  : cart_list.product.image_url,
                } for cart_list in order.cart_set.all()
            ]
```
cart_lists라는 변수가 한 곳에서 밖에 쓰이지 않습니다 :)
가독성을 많이 해치지 않는 범위에서, 꼭 필요하지 않은 변수는 선언하지 않으셔도 좋습니다

Reply:
이 후부터는 리펙토링하면서 
불필요한 부분을 체크하는 습관을 갖도록 해보겠습니다.